gecode/flatzinc.hh

```diff
@@ -484,6 +484,68 @@ namespace Gecode { namespace FlatZinc {
     /// The integer variables used in LNS
     Gecode::IntVarArray iv_lns;
 
+    /* === Experimental `on_restart` support === */
+    class OnRestartHandle : public SharedHandle {
+    protected:
+      class OnRestartData : public SharedHandle::Object {
+        public:
+        /// Marker set to `true` if solve process can be marked as complete
+        bool mark_complete = false;
+        /// Inclusive ranges to assign random values
+        ArgArrayBase<std::pair<int, int>> uniform_range_int;
+        ArgArrayBase<std::pair<FloatVal, FloatVal>> uniform_range_float;
```

Comment:
I think these should be guarded under `GECODE_HAS_FLOAT_VARS`?

Reply:
Yes, I missed the guard there. I will add them when I have some time